app/views/caseload.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+{% include "includes/breadcrumbs.html" %}
+
+{% include "includes/validation-error-messages.html" %}
+
+{% include "includes/title-subtitle.html" %}
+
+{% include "includes/sub_nav.html" %}
+
+<h3 class="heading-medium">Overall caseload by tier</h3>
+
+<table class="sln-table-caseload data-table">
```

Comment:
Could we extra this to a caseload table includes? Just to follow suit from the rest of the main pages.